1-exercises/A-this/exercise-1.js

```diff
@@ -15,8 +15,10 @@ let person = {
     name: "Adrian",
     age: 32,
     currentAddress: "London",
-    celebrateBirthday: /* todo */
-};
+    celebrateBirthday: function () {
+        return this.age;
```

Comment:
Nice one but I think you should add 1 more to age.

Reply:
Thank you so much. 
I am a bit confused about 'add 1 more to age', could you explain for it, please?

Reply:
Yes of course, what is required of us is that  "When called, celebrateBirthday should increment the age of the person by 1."
so you should write like this : "        return this.age + 1;  "